File: Cobalt/src/main/scala/org/squiddev/cobalt/function/UnwindableCallable.scala
package org.squiddev.cobalt.function

import org.squiddev.cobalt.{LuaError, LuaState, LuaThread, UnwindThrowable}
import org.squiddev.cobalt.debug.DebugFrame

trait UnwindableCallable {
  @throws[LuaError]
  @throws[UnwindThrowable]
  def call(state: LuaState, thread: LuaThread, di: DebugFrame, cont: EvalCont): Unit
}

File: Cobalt/src/main/scala/org/squiddev/cobalt/function/UnwindableCallable.java
package org.squiddev.cobalt.function;

import org.squiddev.cobalt.LuaError;
import org.squiddev.cobalt.LuaThread;
import org.squiddev.cobalt.UnwindThrowable;
import org.squiddev.cobalt.debug.DebugFrame;

public interface UnwindableCallable {
	void call(LuaThread thread, DebugFrame di, EvalCont cont) throws LuaError, UnwindThrowable;
}
